solutions/src/main/java/ru/sberbank/school/task07/Test.java

```diff
@@ -0,0 +1,48 @@
+package ru.sberbank.school.task07;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeSet;
+
+public class Test {
+    public static void main(String[] args) {
+        String path = "C:\\Users\\Anastasia\\Desktop\\Java\\Tasks\\homework\\solutions\\src\\main\\java\\"
+                + "ru\\sberbank\\school\\task07\\test.txt";
+        FileParser fileParser = new FileParserImpl();
+        try {
+            List<String> stringsFromFile = fileParser.parse(path);
+            System.out.println(stringsFromFile);
+            Counter counter = new CounterImpl(fileParser);
+            int difWords = counter.count(path);
+            System.out.println("Dif Words: "+difWords);
```

Comment:
WhitespaceAround: &#39;+&#39; is not followed by whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a813224827>*